src/ui/component/textHighlighter/textHighlighter.tsx

```diff
@@ -0,0 +1,27 @@
+import type { FC } from 'react'
+import { splitTextByTerm } from '@/util/splitTextByTerm/splitTextByTerm'
+
+type TextHighlighterProps = {
+  text: string
+  termToHighlight: string
+}
+export const TextHighlighter: FC<TextHighlighterProps> = ({
+  text,
+  termToHighlight: textToHighlight
+}) => {
+  const parts = splitTextByTerm(text, textToHighlight)
+
+  return (
+    <>
+      {parts.map((part, index) =>
+        part.toLowerCase() === textToHighlight.toLowerCase() ? (
```

Comment:
We can provide such option. I do think most of the time the user quickly want to search for a term. He doesn't care, even sometimes doesn't know where are the capitalized letters in the text to search. What do you think?

Reply:
According to the product team, lowercase and uppercase have no impact on searches.

Reply:
Ideally, a comparison function should be provided as part of the props, so that you can "scan" the term to highlight in the text. But the implementation is a bit tricky, from what I can see.

And the default implementation would be the string comparison while ignoring case (of course), so that the user doesn't have to worry about it, most of the time.